bot/cogs/defcon.py

```diff
@@ -139,16 +138,16 @@ def mod_log(self) -> ModLog:
                     'data': {
                         # TODO: retrieve old days count
                         'days': days,
-                        'enabled': enabled,
+                        'enabled': action is not Action.DISABLED,
                     }
                 }
             )
         except Exception as err:
             log.exception("Unable to update DEFCON settings.")
             error = err
         finally:
-            await ctx.send(self.build_defcon_msg(action_type, error))
-            await self.send_defcon_log(action_type, ctx.author, error)
+            await ctx.send(self.build_defcon_msg(action.name.lower(), error))
+            await self.send_defcon_log(action.name.lower(), ctx.author, error)
```

Comment:
Why not just pass the enum object instead of using its name? Then you won't even need the `get_info` for anything.

Reply:
That's very true lol, I still am too tunnel vision into what each functions are doing. That's a great call!